pip/utils/packaging.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import absolute_import
+import logging
+import sys
+
+from pip._vendor.packaging import specifiers
+from pip._vendor.packaging import version
+
+logger = logging.getLogger(__name__)
+
+
+def check_requires_python(requires_python):
+    """
+    Check if the python version in used match the `requires_python` specifier.
+
+    Return `True` if the version of python in use matches the requirement.
+    Return `False` if the version of python in use does not matches the
+    requirement. Raises an InvalidSpecifier if `requires_python` have an
```

Comment:
Maybe put the `Raise` on a new line.
And either put `Return/Raise` or `Returns/Raises` ?